bin/Audio_Transcription.md

```diff
@@ -0,0 +1,38 @@
+## Transcribing longer audio clips
+
+### 0. Pre-requisites
+
+```
+pip3 install --user webrtcvad; pip3 install --user deepspeech; pip3 install --user tensorflow; pip3 install -r requirements.txt
+sudo apt install python3-pyqt5 libcanberra-gtk-module
+
+export PATH=$PATH:~/.local/bin/
+```
+
+### 1. Command line tool
+
+The command line tool processes all the wave files present in the directory mentioned during executing.
```

Comment:
That's not totally accurate, it seems it is going to also perform the task for all models? I'm not so sure about the final goal for that tool, can you elaborate the use case ?

Reply:
So, the command line tool can transcribe a number of wave files present in the directory. We run it with the following inputs:
1. Aggressiveness for VAD
2. Path to a directory containing the wave files 
3. Path to a directory containing the models. So it won't perform the task for all the models, but for the one that we specify only.

Reply:
Ok, please explain me how different that part is from the multi-wav in one directory we have for the C++ client already ? Except the VAD, of course.

To put it another way, I'm wondering if it's a good idea to throw those features in a completely different tool, doing mostly the same as existing ones: it's more code to maintain, to document, more ways for people to get lost.

The fact that you did the PR acknowledge a real need, but maybe it might be addressed by fixing the codebase another way? That also depends on how much time you can allocate, of course :).

Reply:
I wasn't aware of the multi-wav C++ client. I thought the `--audio /path/to/audio/file.wav` took only a single wav file. 
I agree with the "more similar code to maintain". How bout I change the command line tool to do the same thing as the GUI tool. Would that help us out ?

Reply:
https://github.com/mozilla/DeepSpeech/blob/master/native_client/client.cc#L225-L257

Reply:
That could help for sure :)

Reply:
How do you suggest we proceed with this ? 
Have the CMD and GUI tool do the same job, perform VAD and consolidate the output ?

Reply:
Yeah, I'd expect that like it.

Reply:
Cool, I'll do that :)